### ISSUE TITLE
nan for mIOU

### ISSUE THREAD

**Issue by niloofarAzari** [lxtGH/SFSegNets#56]
Executing the eval.py file using pre-trained model "res18_sfnet.pth", i get nan value for mIOU and also for the iou of all the classes.
Although when looking at the output segmented images they seem really good. What may the reason be?

**Comment by lxtGH** [comment]
Could you show the details comand?

**Comment by TiesHendrickx** [comment]
I encountered the same issue. It seems like the loaded GT labels for the test set are all 255 (invalid).
